MessageBird/Client.cs

```diff
@@ -59,7 +62,7 @@ public Message SendMessage(string originator, string body, long[] msisdns, Messa
             var recipients = new Recipients(msisdns);
             var message = new Message(originator, body, recipients, optionalArguments);
 
-            var messages = new Messages(message);
+            var messages = new Resources.Messages(message);
```

Comment:
Can you add a using statement for MessageBird.Resources so you don't have to add the namespace here?